client/components/FooterAccordion.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from "./Footer.module.css";
+
+const FooterAccordion = ( {links} ) => {
+  const linkItems = links.map((link, index) => (
```

Comment:
Accordion is when a section of data needs to be open and closed.  Can't find why it is needed on the footer after examining the footer on Nike detail page.

Reply:
In mobile view (<640px), the foot has two expandable accordions.